tests/e2e/types.go

```diff
@@ -0,0 +1,52 @@
+package e2e
+
+import (
+	"fmt"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+)
+
+// Versions.
+type Version = string
```

Comment:
**Recommended**: IMO we **SHOULD** not separate the type definition of pseudo-enums and their possible values - in this case Local.
It makes tracking things very difficult, I would rather move that to a version.go or leave both const and var in global.go, but definitely not separate OOP types/structs from their values/functions.

Reply:
fixed: [b6683a2](https://github.com/banzaicloud/koperator/pull/987/commits/b6683a24f59cf2bfcfb5978bad12f6800ac94824)